src/mlpack/methods/ann/layer/lstm.hpp

```diff
@@ -87,6 +87,21 @@ class LSTM
   template<typename InputType, typename OutputType>
   void Forward(InputType&& input, OutputType&& output);
 
+  /**
+   * Ordinary feed forward pass of a neural network, evaluating the function
```

Comment:
I think it should be `feed-forward` here.

Reply:
Done